lib/mongo.ex

```diff
@@ -1170,11 +1192,12 @@ defmodule Mongo do
   end
 
   defp normalise_updates([[{_, _} | _] | _] = updates) do
-    updates
-    |> Enum.map(&normalise_update/1)
+    Enum.map(updates, &normalise_update/1)
   end
 
-  defp normalise_updates(updates), do: normalise_updates([updates])
+  defp normalise_updates([{_, _} | _] = updates), do: normalise_updates([updates])
+
+  defp normalise_updates(updates), do: updates
```

Comment:
Only normalise Keyword lists, letting Mongo return an error if the updates are not in an expected format

Reply:
Can you add some names for the placeholders to increase the readability? For example:

Instead of 
```
defp normalise_updates([[{_, _} | _] | _] = updates) do
```

it would be nice to have
```
## maps lists of keywords list
defp normalise_updates([[{_key, _value} | _rest] | _updates] = updates) do
```